src/hotspot/share/jfr/recorder/repository/jfrEmergencyDump.cpp

```diff
@@ -374,19 +374,24 @@ static void write_repository_files(const RepositoryIterator& iterator, char* con
     if (current_fd != invalid_fd) {
       const int64_t size = file_size(current_fd);
       assert(size > 0, "invariant");
-      int64_t bytes_read = 0;
-      int64_t bytes_written = 0;
+      unsigned int bytes_read = 0;
+      unsigned int bytes_written = 0;
```

Comment:
Why have you changed this to a 32-bit type?

Reply:
Changed back to `int64_t`.